### DISCUSSION TITLE
Using the new Mapbox GL JS v3.6.0

### DISCUSSION THREAD

**Discussion by PauloJorgeS** [uber/react-map-gl#2426] [General]
What can I do to use the most updated Mapbox GL?
What are the steps to install and use the [Mapbox GL JS v3.6.0  ](https://github.com/mapbox/mapbox-gl-js/releases/tag/v3.6.0)?

My package.json:

```js 
 "devDependencies": {
  "@types/mapbox-gl": "3.1.0",
  
 },
"dependecies" : {
  "mapbox-gl": "3.4.0",
  "react-map-gl": "^7.1.7",
}
```

**Comment by michaelcgorman** [comment]
`mapbox-gl` v3.5 and above are not supported on `react-map-gl` v7. More info: https://github.com/visgl/react-map-gl/issues/2457

Until `react-map-gl` v8 is ready, you'll have to stick with `mapbox-gl` v3.4.

**Reply by PauloJorgeS** [reply]
Thanks
